app/services/idv/steps/send_link_step.rb

```diff
@@ -32,6 +32,7 @@ def update_document_capture_session_requested_at(session_uuid)
         return unless document_capture_session
         document_capture_session.update!(
           requested_at: Time.zone.now,
+          cancelled_at: nil,
```

Comment:
It took me a while to understand why we need this. I'm guessing this is to handle a situation where the user cancels, but then restarts the hybrid session (with the same or different phone number)? Could we include a test for this?

I guess it probably doesn't matter very much in practice, but it's a bit odd to me that if the user decides to cancel and complete document capture on their desktop, we're still operating on the `DocumentCaptureSession` as being live and valid, despite the fact that it has a `cancelled_at` value. Maybe we could unset this if they reach the document capture step on any device. Or maybe we destroy the session altogether and create a new one ([send them back to welcome step](https://github.com/18F/identity-idp/blob/c530974a81a8eb1bf6a3e3f5b5d380b6873b1ea0/app/services/idv/steps/welcome_step.rb#L3-L6) if polling [reveals there's no document capture session](https://github.com/18F/identity-idp/blob/c530974a81a8eb1bf6a3e3f5b5d380b6873b1ea0/app/controllers/idv/capture_doc_status_controller.rb#L17)?).